s3tests_boto3/functional/test_s3.py

```diff
@@ -7575,6 +7577,284 @@ def test_cors_header_option():
+    _cors_request_and_check_neofs_s3(method='OPTIONS', url=url,
+                                     headers={'Origin': 'http://www.example1.com',
+                                              'Access-Control-Request-Method': 'DELETE'},
+                                     expected_status=200,
+                                     expected_headers={response_origin_header: 'http://www.example1.com', response_methods_header: 'PUT, DELETE'})
+    _cors_request_and_check_neofs_s3(method='OPTIONS', url=url,
+                                     headers={'Origin': 'http://www.example2.com',
+                                              'Access-Control-Request-Method': 'DELETE'},
+                                     expected_status=200, expected_headers={response_origin_header: 'http://www.example2.com', response_methods_header: 'DELETE'})
+    _cors_request_and_check_neofs_s3(method='OPTIONS', url=url,
+                                     headers={'Origin': 'http://www.example2.com',
+                                              'Access-Control-Request-Method': 'PUT'},
+                                     expected_status=403, expected_headers=no_origin_header)
+
+    _cors_request_and_check_neofs_s3(method='OPTIONS', url=url,
+                                     headers={'Origin': 'http://not.exists', 'Access-Control-Request-Method': 'GET'},
+                                     expected_status=200, expected_headers={response_origin_header: '*', response_methods_header: 'GET'})
+    _cors_request_and_check_neofs_s3(method='OPTIONS', url=url,
+                                     headers={'Origin': 'http://not.exists', 'Access-Control-Request-Method': 'PUT'},
+                                     expected_status=403, expected_headers=no_origin_header)
+
+    #with_credentials
+    _cors_request_and_check_neofs_s3(method='GET', url=url, headers={'Origin': 'http://www.example1.com'}, expected_status=None,
+                                     expected_headers={response_origin_header: 'http://www.example1.com'}, with_creds=True)
+    _cors_request_and_check_neofs_s3(method='PUT', url=obj_url, headers={'Origin': 'http://www.example1.com'}, expected_status=None,
+                                     expected_headers={response_origin_header: 'http://www.example1.com'}, with_creds=True)
+    _cors_request_and_check_neofs_s3(method='GET', url=url, headers={'Origin': 'http://not.exists'}, expected_status=None,
+                                     expected_headers={response_origin_header: 'http://not.exists'}, with_creds=True)
+
+
+@attr('fixed_to_neofs_s3_gw')
+@attr(resource='bucket')
+@attr(method='get')
+@attr(operation='check cors response when origin is set to wildcard')
+@attr(assertion='returning cors header')
+@attr('cors-neofs-s3')
+def test_cors_origin_wildcard_neofs_s3():
+    bucket_name = _setup_bucket_acl(bucket_acl='public-read')
```

Comment:
This is just a fixed copy of original test looking like:
```
def test_cors_origin_wildcard():
    bucket_name = _setup_bucket_acl(bucket_acl='public-read')
    client = get_client()

    cors_config ={
        'CORSRules': [
            {'AllowedMethods': ['GET'],
             'AllowedOrigins': ['*'],
            },
        ]
    }

    e = assert_raises(ClientError, client.get_bucket_cors, Bucket=bucket_name)
    status = _get_status(e.response)
    eq(status, 404)

    client.put_bucket_cors(Bucket=bucket_name, CORSConfiguration=cors_config)

    time.sleep(3)

    url = _get_post_url(bucket_name)

    _cors_request_and_check(requests.get, url, None, 200, None, None)
    _cors_request_and_check(requests.get, url, {'Origin': 'example.origin'}, 200, '*', 'GET')
``` 

I suppose we can delete the test because this cases are checked in `test_cors_origin_response_neofs_s3`?